OSC

hi,
I just pulled the latest files today, now I get 8 errors on Osc.cpp: 
Error	236	error C2664: 'std::atomic_bool::atomic_bool(const std::atomic_bool &)' : cannot convert argument 1 from 'bool' to 'const std::atomic_bool &'	C:\cpp\Cinder\blocks\OSC\src\Osc.cpp	line 1006 (same for lines: 1012 1018 1396 1482 1488 1494 1500).
I'm using visual studio 2013.
Thank you
Bruce

--- a/blocks/OSC/src/Osc.h
+++ b/blocks/OSC/src/Osc.h
@@ -617,7 +617,7 @@ class SenderTcp : public SenderBase {
 	asio::ip::tcp::endpoint mLocalEndpoint, mRemoteEndpoint;
 	OnConnectFn				mOnConnectFn;
 	std::mutex				mOnConnectFnMutex;
-	std::atomic_bool		mIsConnected;
+	std::atomic<bool>		mIsConnected;
 	
   public:
 	//! Non-copyable.
@@ -833,7 +833,7 @@ class ReceiverTcp : public ReceiverBase {
 		TcpSocketRef			mSocket;
 		ReceiverTcp*			mReceiver;
 		asio::streambuf			mBuffer;
-		std::atomic_bool		mIsConnected;
+		std::atomic<bool>		mIsConnected;
 		
 		const uint64_t			mIdentifier;
 		
@@ -879,7 +879,7 @@ class ReceiverTcp : public ReceiverBase {
 	using UniqueConnection = std::unique_ptr<Connection>;
 	std::vector<UniqueConnection>		mConnections;
 	uint64_t							mConnectionIdentifiers;
-	std::atomic_bool					mIsShuttingDown;
+	std::atomic<bool>					mIsShuttingDown;
 
 	friend struct Connection;
   public:

